common/utils/dataflow_utils.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package utils contains common helper functions used across multiple other packages.
+// Utils should not import any Spanner migration tool packages.
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"cloud.google.com/go/dataflow/apiv1beta3/dataflowpb"
+	"golang.org/x/exp/maps"
+)
+
+func GetGcloudDataflowCommand(req *dataflowpb.LaunchFlexTemplateRequest) string {
```

Comment:
Should we also call this method and print the equivalent command in the CLI flow?

Reply:
I was thinking that, and it makes sense for non-sharded flow, but for sharded we need to come up with a good way to output it for all shards (in a file maybe? but we already produce like 5 files).

Reply:
I was thinkng lets just write them as debug logs for now. We have `--log-level` flag, if the customer does not want they can set the log level to INFO